app/dashboard/views/layout.ejs

```diff
@@ -1,5 +1,5 @@
 <!DOCTYPE html>
-<html lang="en">
+<html lang="en" style="display: block !important;">
```

Comment:
Why are you adding `display: block` on the html tag?, also why are you doing it inline?